### PR TITLE
Open telemtry

### PR DESCRIPTION
**Pull request by yyyogev**
# What changed
https://deploy-preview-733--logz-docs.netlify.app/shipping/shippers/opentelemetry
moved OpenTelemetry to Shippers > Tracing.

We may change the Shipper tab to ==> Tracing.
Possible limitation on number of tabs that can be displayed in that area. 

Add readme to OpenTelemetry collector with logz.io exporter <== What is this?

- Experiencing problems with top horizontal tabs for Shipping content - can't add tabs for Distributed Tracing or Cloud SIEM shippers to that menu. Code doesn't work...
- to do: remove Jaeger & Zipkin from ship you data > logs > filters >  tracing menu

----

## Pages to review

- https://deploy-preview-733--logz-docs.netlify.app/shipping/log-sources/opentelemetry.html
- https://deploy-preview-733--logz-docs.netlify.app/shipping/

## Remaining work
@YajB You can use this https://deploy-preview-733--logz-docs.netlify.app/shipping/log-sources/elastic-container-service.html as a model. 

- [ ] Adjust all links to open in new tab? 
- [x] Add icon to external links
- [x] Line 24 - clarify. 
- [x] Need an icon on this page: https://deploy-preview-733--logz-docs.netlify.app/shipping/

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/logzio_collections/_log-sources/opentelemetry.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Ship traces with OpenTelemetry to Logz.io
```
Comment (shalper) [inline _source/logzio_collections/_log-sources/opentelemetry.md:2]:
```suggestion
title: Ship traces with OpenTelemetry to Logz.io
logo:
  logofile: cisco-meraki-logo.png
  orientation: horizontal
```

File: _source/logzio_collections/_log-sources/opentelemetry.md [hunk 2]
```diff
@@ -1,48 +1,57 @@
...
 
 The OpenTelemetry Collector pipeline has the following main components: 
+
 * Receivers
 * Processors 
-* Exporters. 
+* Exporters
 
-OpenTelemetry also includes extentions for additional functionality such as diagnostics and health checks. OpenTelemetry has a dedicated collector for contributed components, such as the Logz.io exporter.
+OpenTelemetry also includes extentions for additional functionality such as diagnostics and health checks, as well as a dedicated collector for contributed components, such as the Logz.io exporter.
```
Comment (YajB) [inline _source/logzio_collections/_log-sources/opentelemetry.md:26]: @yyyogev - What are "contributed components?

Comment (yyyogev) [reply _source/logzio_collections/_log-sources/opentelemetry.md:26]: Any components (receivers, processors, exporters, and extensions) in the collector that external source made, like us.

Comment (YajB) [reply _source/logzio_collections/_log-sources/opentelemetry.md:26]: @yyyogev  - could I call them 3rd party components

Comment (yyyogev) [reply _source/logzio_collections/_log-sources/opentelemetry.md:26]: I guess..

**Comment by yyyogev** [comment]
@YajB @shalper are we waiting for something with this one?

**Review by YajB — APPROVED** [review]
edited for clarity and formatting

**Pull request merged**